word_ladder.py

```diff
@@ -12,11 +12,19 @@
 
 words_set = set(words)
 
+words_set_neighbors = gen_neighbors(words)
+
 
 @app.route('/<input_word>', methods=['GET'])
 def serve(input_word):
     print('input_word', input_word)
-    outputs = gen_words(input_word)
+    outputs = []
+
+    if words_set_neighbors[input_word] is None:
```

Comment:
`if not words_set_neighbors[input_word]:`

Reply:
Actually, this would just mean words that dont have any neighbors gets regenerated.

Do this instead:
`if input_word not in words_set_neighbors:`